probability_flavour_text.py

```diff
@@ -8,7 +8,7 @@ def get_flavour(probability):
     elif probability < 10**-21:
```

Comment:
Ideally 1 PR per problem (fixing kree table, and fixing typo), but this is fine for now.